constance/backends/memory.py

```diff
@@ -0,0 +1,35 @@
+from threading import Lock
+
+from . import Backend
+from .. import signals, config
+
+
+class MemoryBackend(Backend):
+    """
+    Simple in-memory backend that should be mostly used for testing purposes
+    """
+    def __init__(self):
+        super().__init__()
+        self._storage = {}
```

Comment:
I'm probably a bit uncertain about threading internals, but I think we should move this into a module global so separate memory backend instances in the individual threads are using the same global storage, since that's the use case of constance.

Reply:
ok, I've got your point, will make corresponding updates